src/NuGet/Microsoft.CodeAnalysis.CSharp.CodeStyle.nuspec

```diff
@@ -33,6 +33,60 @@
     <file src="Dlls\CodeStyleFixes\Microsoft.CodeAnalysis.CodeStyle.Fixes.dll" target="analyzers\dotnet\cs" />
     <file src="$thirdPartyNoticesPath$" target="" />
 
+    <!-- Satellite assemblies -->
+    <file src="Dlls\CSharpCodeStyle\cs\*" target="analyzers\dotnet\cs\cs" />
```

Comment:
If I'm reading https://docs.microsoft.com/en-us/nuget/schema/nuspec right, this should allow you to not have to spell out each language:

``` xml
<file src="Dlls\CSharpCodeStyle\**\*.resources.dll" target="analyzers\dotnet\cs" />
```

Aside: are there any plans to move to the msbuild /t:pack instead of hand-crafted nuspecs?

Reply:
Yes. At some point :)

Reply:
BTW, msbuild /t:pack doesn't support tools and analyzer packages (there is some support for tools packages, but it's not sufficient). So some packages will still need hand-crafted nuspecs.

Reply:
I'm going to pull the .nuspec changes from the PR and create a separate PR from them after the loc team has done their initial pass. That way we're not producing NuGet packages with "localized" assemblies that aren't really localized. When I create that new PR I'll go back and use wildcards.

Reply:
Good call.